### PR TITLE
Add Scala-lang.org.xml (fix #15017)

### LINKED ISSUE #15017: A rule for *.scala-lang.org
Type: new ruleset
Domain: *.scala-lang.org

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: src/chrome/content/rules/Scala-lang.org.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Scala-lang.org">
+	<target host="scala-lang.org" />
+	<target host="www.scala-lang.org" />
+	<target host="contributors.scala-lang.org" />
+	<target host="days2010.scala-lang.org" />
+	<target host="days2010.scala-lang.org" />
```
Comment (cschanaj) [inline src/chrome/content/rules/Scala-lang.org.xml]: `days2010` appeared twice

File: src/chrome/content/rules/Scala-lang.org.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<!--
+	Nonfunctional hosts in *.scala-lang.org:
+		- codereview.scala-lang.org (404)
```
Comment (Bisaloo) [inline src/chrome/content/rules/Scala-lang.org.xml]: It's not clear what's the issue is here. It should probably be changed to something such as `Different content HTTP/HTTPS`

File: src/chrome/content/rules/Scala-lang.org.xml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Scala-lang.org">
+	<target host="scala-lang.org" />
+	<target host="www.scala-lang.org" />
+	<target host="contributors.scala-lang.org" />
+	<target host="docs.scala-lang.org" />
+	<target host="index.scala-lang.org" />
+	<target host="issues.scala-lang.org" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Scala-lang.org.xml]: packages.scala-lang.org

File: src/chrome/content/rules/Scala-lang.org.xml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+	<target host="platform.scala-lang.org" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Scala-lang.org.xml]: scaladex.scala-lang.org

File: src/chrome/content/rules/Scala-lang.org.xml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+	<target host="scastie.scala-lang.org" />
+	<target host="users.scala-lang.org" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Scala-lang.org.xml]: wiki.scala-lang.org

**Comment by J0WI** [comment]
All remaining hosts are added, but chrome beta tests are currently broken.

**Pull request merged**